ui/src/app/registration/modal/modal.component.ts

```diff
@@ -96,30 +102,31 @@ export class RegistrationModalComponent implements OnInit {
   private getForm(role: string): FormGroup {
     if (role === 'installer') {
       return this.formBuilder.group({
-        companyName: new FormControl("", Validators.required),
-        firstname: new FormControl("", Validators.required),
-        lastname: new FormControl("", Validators.required),
-        street: new FormControl("", Validators.required),
-        zip: new FormControl("", Validators.required),
-        city: new FormControl("", Validators.required),
-        country: new FormControl("", Validators.required),
-        phone: new FormControl("", Validators.required),
-        email: new FormControl("", [Validators.required, Validators.email]),
-        password: new FormControl("", Validators.required),
-        confirmPassword: new FormControl("", Validators.required),
+        companyName: new FormControl("", [Validators.required, Validators.maxLength(50)]),
+        firstname: new FormControl("", [Validators.required, Validators.maxLength(50)]),
+        lastname: new FormControl("", [Validators.required, Validators.maxLength(50)]),
+        street: new FormControl("", [Validators.required, Validators.maxLength(50)]),
```

Comment:
I am not sure we should be so strict. There are again a lot of discussions on that topic on the internet (e.g. https://ux.stackexchange.com/questions/55529/what-should-the-character-limits-for-first-last-name-inputs-be)

While I think it is indeed feasible to have a limit, I think 50 characters could be too limiting in a few cases practice. How about 100?